spec/support/capybara_helpers.rb

```diff
@@ -21,92 +21,99 @@ def expect_css(css)
 
-  def expect_select2_selection(text)
-    expect(page).to have_css(".select2-selection__rendered", text: /#{text}/)
+  def slimselect_addable_button
+    slimselect_popover.find(".ss-addable")
   end
 
-  def expect_select2_empty_selection
-    expect(page).not_to have_css(".select2-selection__rendered")
+  def add_slimselect_option(item_text)
+    fill_slimselect_input(item_text)
+    slimselect_addable_button.click
   end
 
-  def expect_select2_choices_count_to_eq(count)
-    expect(page).to have_css("li.select2-selection__choice", count: count)
+  def click_slimselect_option(option_text, id = nil)
+    slimselect_popover(id).find(".ss-option", text: option_text).click
   end
 
-  def expect_select2_options_count_to_eq(count)
-    expect(page).to have_css("select.select2-hidden-accessible option", count: count)
+  def pick_slimselect_entered_option(item_text, display_name = nil)
+    display_name = item_text unless display_name
+    fill_slimselect_input(item_text)
+    click_slimselect_option(display_name)
   end
 
-  def expect_select2_result_text_to_eq(result_number, text)
-    expect(page).to have_css(
-      "li.select2-results__option:nth-child(#{result_number})", text: /#{text}/
-    )
+  def expect_slimselect_data_option(option, value)
+    expect(page).to have_xpath("//select[@data-#{option}='#{value}']")
   end
 
-  def expect_select2_results_count_to_eq(count)
-    klass = "li.select2-results__option"
-    no_results = "No results found"
+  def expect_slimselect_selection(text)
+    expect(page).to have_css(".ss-main", text: /#{text}/)
+  end
 
-    if count.zero?
-      expect(page).to have_css(klass, count: 1)
-      expect(page).to have_content(no_results)
-    else
-      expect(page).to have_css(klass, count: count)
-      expect(page).not_to have_content(no_results)
-    end
+  def expect_slimselect_empty_selection
+    expect(page).to have_css(".ss-main", text: /Select Value/)
   end
 
-  def click_add_nested
-    find("a.has_many_add").click
+  def expect_slimselect_result_text_to_eq(result_number, text)
+    expect(page).to have_css(
+      "div.ss-option:nth-child(#{result_number})", text: /#{text}/
+    )
   end
 
-  def on_nested_ctx(resource_number, &block)
-    within("li.has_many_container fieldset:nth-child(#{resource_number + 1}) ") do
-      block.call
-    end
+  def expect_slimselect_error(text)
+    expect(page).to have_css(".ss-error", text: text)
   end
 
-  def expect_nested_select2_result_text_to_eq(result_number, text)
+  def expect_nested_slimselect_result_text_to_eq(result_number, text)
     expect(page).to have_css(
       "li.nested_level:nth-child(#{result_number})", text: /#{text}/
     )
   end
+
+  def close_slimselect_options(id = nil)
+    slimselect_search_input(id || slimselect_original_select_id).native.send_keys(:escape)
+  end
+
+  def expect_slimselect_popover_is_closed(id = nil)
+    expect(page.document).not_to have_css(".ss-content[data-id=#{id || slimselect_original_select_id}]")
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is too long. [104/100]